libmamba/tests/src/core/test_history.cpp

```diff
@@ -26,9 +26,9 @@
 
 namespace mamba
 {
-    TEST_SUITE("history")
+    namespace
     {
-        TEST_CASE("parse")
+        TEST_CASE("History parse")
```

Comment:
@mathbunnyru Pinging here in case the comment was missed as we are planning to merge this soon. Thanks!

Reply:
> > I guess the same thing might have been true for doctest, but there were TEST_SUITEs, that's why they were considered not the same
> 
> Actually I am wondering here if we shouldn't give a name to the `namespace` replacing `TEST_SUITE` instead of an anonymous one. That would make it more consistent with what we had before and will give a hint on what we are testing just like the purpose of `TEST_SUITE`. I was also wondering how we would filter out tests with `catch2` to run them individually? Something equivalent to: `./test -ts=testsuite_name -tc=testcase_name -sc=subcase_name`

I don't think having a proper namespace name will help in any way.
It is currently not consistent at all, so I would probably just remove it completely, in a separate commit, rather than have it in a few places.

The reason it doesn't really help is because there are many ways to run Catch2 tests.
I will show it with an example of `TEST_CASE("HTTPMirror")` located in `test_mirror.cpp` file.

1. `$TEST_BIN HTTPMirror` will run this test case with all 3 SECTIONs
2. `$TEST_BIN --section https HTTPMirror` will only run `https` SECTION
3. `$TEST_BIN --filenames-as-tags [#test_mirror]` will run all the files in the `test_mirror.cpp` file
4. You can combine these options, for example `$TEST_BIN --filenames-as-tags --section https [#test_mirror] HTTPMirror`. Note that for example `$TEST_BIN --filenames-as-tags --section https [#test_mirror]` will run sections named `https`, but also `REQUIRE`s which are outside of `SECTION`s (all that only in `test_mirror.cpp` file).

From what I saw most of the time `TEST_SUITE` was surrounding the whole file.
But we don't need to do that - we can just use the test filename.

Hope this helps.